src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+
+    @Override
+    public void doWork() {
+        System.out.println("Bulldozer started working");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Bulldozer stopped working");
```

Comment:
Similar to the doWork method, the stopWork message 'Bulldozer stopped working' could be more informative. Make sure it indicates clearly that it is the Bulldozer that has stopped working.